src/InteractiveWindow/Editor/InteractiveWindow.cs

```diff
@@ -1885,6 +1822,10 @@ private void StoreUncommittedInput()
 
         private void AppendUncommittedInput(string text)
         {
+            if (string.IsNullOrEmpty(text))
```

Comment:
I don't think it can be an assert because the value comes in through a public API and isn't validated.  There should, however, be an ```else``` before the following ```if```.